### ISSUE TITLE
STM32L4: Device file wrong

### ISSUE THREAD

**Issue by strongly-typed** [roboterclubaachen/xpcc#239]
In the STM32L4 [device file](https://github.com/roboterclubaachen/xpcc/blob/develop/src/xpcc/architecture/platform/devices/stm32/stm32l476_486-j_m_q_r_v_z-c_e_g.xml#L196) has some issues with the `BreakIn`.
```xml
      <gpio port="A" id="6">
        <af id="1" peripheral="Timer1" name="BreakIn" type="in"/>
        <af id="12" peripheral="Timer1" name="BreakIn" type="in"/>
      </gpio>
```

The generated `gpio.hpp` won't compile because the connect method is overloaded with the same `TypeId`. Your third easter egg ;-)

There are many other occurrences of that bug in that file.

There seems to be an aliasing between different signals to the same name in the parsing of the device files.

<img width="980" alt="screen shot 2017-04-08 at 14 03 36" src="https://cloud.githubusercontent.com/assets/1626990/24829172/4ff1c8a6-1c6d-11e7-9aa4-7c92516fe652.png">

<img width="1007" alt="screen shot 2017-04-08 at 14 02 51" src="https://cloud.githubusercontent.com/assets/1626990/24829192/a4876ba0-1c6d-11e7-89b0-cc7c7b80a135.png">

@salkinium Probably that can only be fixed by you.

**Comment by salkinium** [comment]
AF12 should be named `BreakInComp2`. Unfortunately the design of `connect` is broken, because the data doesn't mold itself to our naming abstraction. The place to hotfix that [in the DFG would be here](https://github.com/roboterclubaachen/xpcc/blob/develop/tools/device_file_generator/dfg/stm32/stm_reader.py#L377-L378).

I don't see this truly being fixed in xpcc though. I would recommend removing connect completely and making the `setAlternateFunction()` method public as well as implement a basic pinmux HAL for the F1/L1 series, similarly to how the `SystemClock` is backed by `xpcc::stm32::Clock`. This relates to https://github.com/roboterclubaachen/xpcc/issues/178#issuecomment-262942020.

**Comment by strongly-typed** [comment]
I finally [fixed it manually in the device file](https://github.com/roboterclubaachen/xpcc/pull/240/commits/bf65099381b9c4c09f8325fcd1c32b54dd47075a). @salkinium Please take care not to overwrite it when updating the device files.

**Comment by salkinium** [comment]
This was fixed in #240.
